docs/dashboard_widget_field_rules.md

```diff
@@ -0,0 +1,81 @@
+# Rules for Adding Fields to Dashboard Widgets
+
+## Before Making Changes
+1. Check if the field is already implemented in the widget's schema and build functions
```

Comment:
```suggestion
1. Check if the field is already implemented in the widget's schema and build functions.
```
Each item in these lists is a full sentence, so typically each one would end with a period, as shown in the suggestions. I won't make suggestions for all of them though, just to avoid cluttering your PR.